src/test/java/com/sleekbyte/tailor/utils/CharFormatUtilTest.java

```diff
@@ -88,4 +88,16 @@ public void testStartsWithAcronym() {
         assertFalse(CharFormatUtil.startsWithAcronym("xURLS"));
         assertFalse(CharFormatUtil.startsWithAcronym("shieldPROGRAMMEmarvel"));
     }
+
+    @Test
+    public void testBacktickEscapedIdentifier() {
+        // Backtick(s) are not part of the identifier
+        assertTrue(CharFormatUtil.unescapeIdentifier("``").isEmpty());
+        assertTrue(CharFormatUtil.unescapeIdentifier("").isEmpty());
+        assertTrue(CharFormatUtil.unescapeIdentifier("`s`").equals("s"));
```

Comment:
you can use ``assertEquals(CharFormatUtil.unescapeIdentifier("`s`"), "s")`` instead (this applies to everything that follows)